pubspec.yaml

```diff
@@ -57,3 +58,8 @@ flutter:
         ffiPlugin: true
       windows:
         ffiPlugin: true
+
+  assets:
```

Comment:
Here I am not sure what to do. These assets are only needed on the web. Is there a way to do not include them on other platforms?

Reply:
I'm not sure, but I think for that we would need a federated plugin (https://docs.flutter.dev/packages-and-plugins/developing-packages#federated-plugins). That's quite a lot of additional work, though. 

(There are open issues on conditional bundling of assets https://github.com/flutter/flutter/issues/65065 and platform specific assets https://github.com/flutter/flutter/issues/8230. I suggest we put a big 👍 on those issues.)

For now I think it's okay to leave everything here? We can tally the size of the additional files and ask people: is web support worth the additional X megabytes on all other platforms, or do you want to wait for us to do it "properly" first?

Reply:
These assets take 626 Kb uncompressed. It's ok also for me to leave everything here. I just add a comment for those 2 issues to remember this.